source/common/unified_memory_allocation/include/uma/memory_pool.h

```diff
@@ -0,0 +1,95 @@
+/*
+ *
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+
+#ifndef UMA_MEMORY_POOL_H
+#define UMA_MEMORY_POOL_H 1
+
+#include <uma/base.h>
+#include <uma/memory_pool_ops.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct uma_memory_pool_t *uma_memory_pool_handle_t;
+
+///
+/// \brief Creates new memory pool
+/// \param ops instance of uma_memory_pool_ops_t
+/// \param params pointer to pool-specific parameters
+/// \return UMA_RESULT_SUCCESS on success or appropriate error code on failure
+///
+enum uma_result_t umaPoolCreate(struct uma_memory_pool_ops_t *ops, void *params,
+                                uma_memory_pool_handle_t *hPool);
+
+///
+/// \brief Destroys memory pool
+/// \param hPool handle to the pool
+///
+void umaPoolDestroy(uma_memory_pool_handle_t hPool);
+
+///
+/// \brief Allocates size bytes of uninitialized storage of the specified hPool
+/// \param hPool specified memory hPool
+/// \param size number of bytes to allocate
+/// \return Pointer to the allocated memory
+///
+void *umaPoolMalloc(uma_memory_pool_handle_t hPool, size_t size);
+
+///
+/// \brief Allocates size bytes of uninitialized storage of the specified hPool
+/// with
```

Comment:
is this an extra line break that we can remove?

Reply:
Sure, done, this was just some clang-format quirk